content/zh-tw/cluster.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 叢集
+status: Completed
+category: 概念
+tags: ["基礎架構", "架構", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+一座叢集是指一組由計算機或應用程式為實現共同目標而協同工作。
+在雲原生計算的背景之下，這個詞彙通常用於 Kubernetes。
+一座 [Kubernetes](/zh-tw/kubernetes/) 叢集是一組執行在它們自己的容器中的服務 (或工作負載)，通常在不同的機器上執行。
```

Comment:
```suggestion
一座 Kubernetes 叢集是一組執行在它們自己的容器中的服務 (或工作負載)，通常在不同的機器上執行。
```
Is it a hyperlink that directly refers to the original text?

Reply:
The link will connect you to the subsequent page dedicated to authoring Kubernetes translations.

https://glossary.cncf.io/cluster/

Reply:
![image](https://github.com/cncf/glossary/assets/64598482/d6deda13-ff3d-4687-9558-650dce2ee5d7)

Links are not used in the original text, but are used in Zh-tw.

Reply:
I don't quite understand why the link position should be changed so that the Zh-tw content does not match the original text

Reply:
I got the point